includes/class-sensei-frontend.php

```diff
@@ -1174,6 +1174,8 @@ public function sensei_course_start() {
 			&& isset( $_POST['course_start'] )
 			&& wp_verify_nonce( $_POST['woothemes_sensei_start_course_noonce'], 'woothemes_sensei_start_course_noonce' )
 			&& Sensei_Course::can_current_user_manually_enrol( $post->ID )
+			&& Sensei_Course::is_prerequisite_complete( $post->ID )
+			&& ! post_password_required( $post->ID )
```

Comment:
Yes. [`post_password_required`](https://developer.wordpress.org/reference/functions/post_password_required/) returns

> `false` if a password is not required or the correct password cookie is present, true otherwise

so if user has not provided password, then `post_password_required()` returns true. I was a bit confused by it and used it incorrectly initially. 😄

Reply:
Yeah their logic seems to work in the opposite direction but okay, what can we do 😆